File: DaftarTim.java
public class DaftarTim {
    private String namaTim;
    private String negaraAsal;
    private int tahunBerdiri;
    private String pemain;
    private int nomor;
    private String pemain2;
    private int nomor2;

    public DaftarTim(String namaTim, String negaraAsal, int tahunBerdiri, String pemain, int nomor, String pemain2, int nomor2){
        this.namaTim = namaTim;
        this.negaraAsal = negaraAsal;
        this.tahunBerdiri = tahunBerdiri;
        this.pemain = pemain;
        this.nomor = nomor;
        this.pemain2 = pemain2;
        this.nomor2 = nomor2;
    }

    public DaftarTim(){
    }

    // Nama Tim
    public String getNamaTim(){
        return namaTim;
    }

    public void setNamaTim(String namaTim){
        this.namaTim = namaTim;
    }

    // Negara Asal
    public String getNegaraAsal(){
        return negaraAsal;
    }

    public void setNegaraAsal(String negaraAsal){
        this.negaraAsal = negaraAsal;
    }

    // Tahun Berdiri
    public int getTahunBerdiri(){
        return tahunBerdiri;
    }

    public void setTahunBerdiri(int tahunBerdiri){
        this.tahunBerdiri = tahunBerdiri;
    }

    // Pemain
    public String getPemain(){
        return pemain;
    }

    public void setPemain(String pemain){
        this.pemain = pemain;
    }

    public String getPemain2(){
        return pemain2;
    }

    public void setPemain2(String pemain2){
        this.pemain2 = pemain2;
    }

    // Pemain
    public int getNomor(){
        return nomor;
    }

    public void setNomor(int nomor){
        this.nomor = nomor;
    }

    public int getNomor2(){
        return nomor2;
    }

    public void setNomor2(int nomor2){
        this.nomor2 = nomor2;
    }
}

File: Latihan1B-py/DaftarTim.py
class DaftarTim:
    __namaTim = ""
    __negaraAsal = ""
    __tahunBerdiri = 0
    __pemain = ""
    __pemain2 = ""
    __nomor = 0
    __nomor2 = 0

    def __init__(self, namaTim, negaraAsal, tahunBerdiri, pemain, nomor, pemain2, nomor2):
        self.__namaTim = namaTim
        self.__negaraAsal = negaraAsal
        self.__tahunBerdiri = tahunBerdiri
        self.__pemain = pemain
        self.__pemain2 = pemain2
        self.__nomor = nomor
        self.__nomor2 = nomor2
    
    # Nama Tim
    def setNamaTim(self, namaTim):
        self.__namaTim = namaTim

    def getNamaTim(self):
        return self.__namaTim
    
    # Negara Asal
    def setNegaraAsal(self, negaraAsal):
        self.__negaraAsal = negaraAsal

    def getNegaraAsal(self):
        return self.__negaraAsal

    # Tahun Berdiri
    def setTahunBerdiri(self, tahunBerdiri):
        self.__tahunBerdiri = tahunBerdiri

    def getTahunBerdiri(self):
        return self.__tahunBerdiri

    # Pemain
    def setPemain(self, pemain):
        self.__pemain = pemain

    def getPemain(self):
        return self.__pemain
    
    def setPemain2(self, pemain2):
        self.__pemain2 = pemain2

    def getPemain2(self):
        return self.__pemain2

    # Nomor
    def setNomor(self, nomor):
        self.__nomor = nomor

    def getNomor(self):
        return self.__nomor
    
    def setNomor2(self, nomor2):
        self.__nomor2 = nomor2

    def getNomor2(self):
        return self.__nomor2